docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -358,6 +358,52 @@ Networking
 
 In general, if you want to use Airflow locally, your DAGs may try to connect to servers which are running on the host. In order to achieve that, an extra configuration must be added in ``docker-compose.yaml``. For example, on Linux the configuration must be in the section ``services: airflow-worker`` adding ``extra_hosts: - "host.docker.internal:host-gateway"``; and use ``host.docker.internal`` instead of ``localhost``. This configuration vary in different platforms. Please check the Docker documentation for `Windows <https://docs.docker.com/desktop/windows/networking/#use-cases-and-workarounds>`_ and `Mac <https://docs.docker.com/desktop/mac/networking/#use-cases-and-workarounds>`_ for further information.
 
+Debug Airflow inside docker container using Pycharm
+===================================================
+Prerequisites: Create a project in **PyCharm** and download the `docker-compose.yaml <{{ doc_root_url }}docker-compose.yaml>`__.
+
+Steps:
+
+1) Modify  ``docker-compose.yaml``
+
+   Add the following section under the ``services`` section:
+
+.. code-block:: docker
+
+    services:
```

Comment:
I'm sure that I know that this line shouldn't be copied.
But worried that end users copy it under services with preserving this line, e.g.

```yaml
services:
  ..
  services:
      airflow-python:
        <<: *airflow-common
        profiles:
          - debug
        environment:
          <<: *airflow-common-env
        user: "50000:0"
        entrypoint: ["bash"]
```

Reply:
I'll make the changes you suggested.